build.sbt

```diff
@@ -0,0 +1,50 @@
+inThisBuild(List(
+  organization := "com.dwolla",
+  description := "A library for streaming healthy service instances in Consul to the RSocket Java load balancer.",
+  homepage := Some(url("https://github.com/Dwolla/rsocket-consul-java")),
+  licenses += ("MIT", url("http://opensource.org/licenses/MIT")),
+  developers := List(
+    Developer(
+      "dwolla",
+      "Dwolla Dev Team",
+      "dev+github@dwolla.com",
+      url("https://dwolla.com")
+    ),
+  ),
+  startYear := Option(2019),
+
+  githubWorkflowJavaVersions := Seq("adopt@1.8", "adopt@1.11"),
+  githubWorkflowTargetTags ++= Seq("v*"),
+  githubWorkflowPublishTargetBranches :=
+    Seq(RefPredicate.StartsWith(Ref.Tag("v"))),
+  githubWorkflowPublish := Seq(
+    WorkflowStep.Sbt(
+      List("ci-release"),
+      env = Map(
+        "PGP_PASSPHRASE" -> "${{ secrets.PGP_PASSPHRASE }}",
+        "PGP_SECRET" -> "${{ secrets.PGP_SECRET }}",
+        "SONATYPE_PASSWORD" -> "${{ secrets.SONATYPE_PASSWORD }}",
+        "SONATYPE_USERNAME" -> "${{ secrets.SONATYPE_USERNAME }}"
+      )
+    )
+  ),
+))
+
+lazy val `rsocket-consul-java` = (project in file("."))
+  .settings(
+    libraryDependencies ++= {
+      val rsocketVersion = "1.0.0-RC1"
```

Comment:
I reverted this back to the version we were previously using, instead of going with the latest `1.0.x`, because @atoivonen had some issues when he tried testing this PR locally. Hopefully this resolves the issue and we can push off the upgrade for another day.